docpages/example_programs/music_and_audio/join_voice.md

```diff
@@ -65,32 +61,30 @@ int main(int argc, char const *argv[])
 				/* We are now connecting to a vc. Wait for on_voice_ready 
 				 * event, and then send the audio within that event:
 				 * 
-				 * event.voice_client->send_audio_raw(...);
+				 * event.voice_client->send_audio_raw(…);
```

Comment:
Yeah the unicode three dots thing DEFINITELY should not be in here.

Reply:
It's still docs, and that part should be replaced if used in actual code anyway. But as already stated, while correct typography, if not liked, can be removed.